### PR TITLE
Kr01 connect to db

### PR DESCRIPTION
**Pull request by s-kyle-reed**
Code to connect to a remote DB via TCP. It also creates a 'Words' class that is used to create a 'Words' table with Entity framework via migrations (these are done via powershell. These commands were found here: https://elanderson.net/2018/09/entity-framework-core-2-1-data-seeding)

### CODE REVIEW

**Comment by s-kyle-reed** [comment]
I got these same conflicts on my local Git repo. Honestly I have no idea what they are.. :(.

File: BingoTestClassLibrary/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace BingoTestClassLibrary
```
Comment (Marina-Yesakova) [inline BingoTestClassLibrary/Class1.cs]: Why do we need this empty class?

File: BingoTestClassLibrary/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+{
+    public class Class1
+    {
+    }
```
Comment (marina-berg) [inline BingoTestClassLibrary/Class1.cs:7]: Why do we need this empty class?

**Review by marina-berg — CHANGES_REQUESTED** [review]
Please publish new iteration after removing bin, obj and vs files. 

1) figure out how to revert your commits, so that your changes are unstaged
2) "git checkout -- vs" 
3) "git checkout -- obj" 
4) "git checkout -- bin" 
5) "git pull origin master"
6) "git commit"
7) "git push"

It's important because the PR is very big and hard to review. Also, we don't have bin, obj and vs folders in the master anymore. And you will get rid of the conflicts that you have now in your PR. Thanks!

**Comment by s-kyle-reed** [comment]
Closing this request to get over these blockers. Will create new pull request and apply changes there

**Comment by marina-berg** [comment]
Sounds good, looking forward to the new PR!

**Pull request closed without merging**